.github/workflows/test.yml

```
@@ -0,0 +1,34 @@
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [17]
+    outputs:
+        test_status: ${{ steps.test_status.outputs.status }}
+    name: Run All unit tests on Java ${{ matrix.java }}
+    steps:
+      - uses: actions/checkout@master
+
+      - name: Setup java
+        uses: actions/setup-java@v1
```

actionlint (1.7.4)

the runner of "actions/setup-java@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)